app/vendors/shells/sphinx_conf.php

```diff
@@ -251,6 +251,55 @@ function($v) { return $v != 'U+300..U+36F'; }
             )
         ))."
         ignore_chars = U+AD, U+301\n";
+        
+        /* Turkish
+         *   Vowels:
+         *     Fold dotless capital I into lowercase dotless i ('I->U+131')
+         *     Fold dotted capital I into lowercase dotted i ('U+130->i')
+         *     Fold {A,a,U,u} + ^ (U+C2, U+E2, U+DB, U+FB) into letters without ^
+         *     Fold {I,i} + ^ (U+CE, U+EE) into lowercase dotless i without ^
+         *     Fold O + diaeresis (U+D6) into o + diaeresis (U+F6)
+         *   Consonants:
+         *     Fold C + cedilla (U+C7) into c + cedilla (U+E7)
+         *     Fold G + breve (U+11E) into g + breve (U+11F)
+         *     Fold S + cedilla (U+15E) into s + cedilla (U+15F)
+         *   Combining marks:
+         *     Ignore combining circumflex (U+302)
```

Comment:
The following code does not ignore U+302. Either it needs to be added to the `ignore_chars` option, or that comment needs to be removed.